features/step_definitions/comments_steps.rb

```diff
@@ -0,0 +1,21 @@
+Given("I have created character and comment") do
+  FactoryBot.create(:character,
+                    :user => @registered_user,
+                    :name => "Wizzard",
+                    :description => "My first wizzard")
+
+  visit user_path(@registered_user)
+  click_on "Wizzard"
+  click_on "New Comment"
+  fill_in "comment_description", :with => "My first comment"
+  click_on "Submit Comment"
+end
+
+When("I visit character Wizzard show page") do
+  visit user_path(@registered_user)
+  click_on "Wizzard"
+end
```

Comment:
You already have "I visit character Trol show page" step, make one shared step instead of those two.